File: database/src/main/java/com/example/database/dtos/AppUserBasicDataDTO.java
package com.example.database.dtos;

import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

import java.util.UUID;

@Value
@Builder
@Jacksonized
public class AppUserBasicDataDTO {
    UUID userId;

    String firstName;

    String lastName;

    String email;

    String username;
}

File: MyApplication/app/src/main/java/com/example/myapplication/dtos/AppUserBasicDataDTO.kt
package com.example.myapplication.dtos

import com.example.myapplication.model.Activity

data class AppUserBasicDataDTO(
    val firstName: String? = null,
    val lastName: String? = null,
    val email: String? = null,
    val lastActivity: Activity? = null,
    val username: String? = null
)